src/core/public/fatal_errors/fatal_errors_screen.test.tsx

```diff
@@ -51,6 +51,7 @@ beforeAll(() => {
   delete window.addEventListener;
 
   window.addEventListener = jest.fn();
+  // @ts-ignore
   window.location = {
```

Comment:
nit, you cast below `window` to the right type, so you don't have to disable TypeScript for the whole line:

```suggestion
  (window as any).location = {
```